salt/modules/cmdmod.py

```diff
@@ -2653,16 +2654,25 @@ def shells():
     return ret
 
 
-def shell_info(shell):
+def shell_info(shell, **kwargs):
```

Comment:
We've been actively engaged in trying to correct cases where `**kwargs` is used in other execution modules.